### DISCUSSION TITLE
Projects Metadata

### DISCUSSION THREAD

**Discussion by emanueledf** [marcusolsson/obsidian-projects#832] [General]
ciao, thanks for the plugin , really useful.  I've a question; I checked the previous discussions but I don't think this was mentioned. 
Is there a reason why project details goes into *\.obsidian\plugins\obsidian-projects\data.json?

I would have expected project related data to be saved into some project directory/note.
Thanks for your work.
Em

**Comment by Acylation** [comment]
That requires extra design on "single file mode", something like kanban or canvas. Noted down as a request and backlogged. Thanks!
